### ISSUE TITLE
brightness does  not return to value set by the function keys on battery

### ISSUE THREAD

**Issue by g7alt** [mate-desktop/mate-power-manager#187]
On Mate 1.12, Linux Mint 17.3, and Ubuntu MATE 16.04
1. When on battery screen brightness drop to 50%, whick is okay.
2. After that I increase brightness with function keys to say 80%, which works ok.
3. After the screen dims, it returns back to 50% again but not 80% ie: the value set by the function keys.
4. If I disable "reduce backlight brightness" on battery and set brightness with the command 
   `gsettings set org.mate.power-manager brightness-ac 80`
   then after dimming it returns to the value I set. So I think it reads the brightness-ac value when to restore brightness after dimming. So function keys aren't writing the brightness value at all.
##

**Comment by snugglej** [comment]
I have the same problem but I also have a backlight on my keyboard. When I turn off the backlight on the keyboard and the system goes to dim the screen using the mate power-manager and then un-dim it will cause the backlight on the keyboard to also come back to the powered on state value of 1 even if it was off or at the state value of 2.

**Comment by narasab** [comment]
Checked out Linux Mint 18 with Mate 1.14, same behaviour there too. This is a critical feature on a laptop.

**Comment by quite** [comment]
I am here for the keyboard backlight issue. I could not find any way to turn it off in the Mate control panel, but did it the hard way by `echo 0 >/proc/acpi/ibm/kbdlight` (on this Thinkpad T450s). But, it doesn't stick. Kbdlight goes back on again typically after graphical login, after re-login to locked screen, and after activating an idle-dimmed screen.

**Comment by simonsaysthis** [comment]
@quite I use a Thinkpad T460s, don't you also have the issue with brightness. I am facing both, keyboard backlight and brightness. I wish somebody had a permanent fix for this so I could use Mate instead of Unity

**Comment by quite** [comment]
Quite possibly I would, but I don't use any kind of feature that dims brightness (on battery mode or otherwise), and I tend not to adjust the brightness in any manual way. (I just have DPMS turn off the display when 30m or so idle).

For reference, this issue is also discussed here:

https://bugzilla.redhat.com/show_bug.cgi?id=1348156
https://bugs.launchpad.net/dell-sputnik/+bug/1510344

**Comment by simonsaysthis** [comment]
I wish somebody would look at this and provide a fix. I have submitted bug requests but they are not being confirmed.

**Comment by narasab** [comment]
This bug is still valid on Mate 1.18.

**Comment by simonsaysthis** [comment]
Yeah. The gift that keeps on giving

**Comment by mladenradev** [comment]
Mint 19 Mate - issue still there....

**Comment by simonsaysthis** [comment]
I don't think this will ever be fixed, to be honest. Been like that for years now.
